flycheck.el

```diff
@@ -7722,8 +7722,8 @@ See URL `https://github.com/commercialhaskell/stack'."
             source)
   :error-patterns
   ((warning line-start (file-name) ":" line ":" column ":"
-            (or " " "\n    ") "Warning:"
-            (optional " " "[" (id (one-or-more not-newline)) "]")
+            (or " " "\n    ") (in "Ww") "arning:"
+            (optional " " "[" (id (one-or-more not-newline)) "] ?")
```

Comment:
Probably weird interactions before the various `optional`. If the whitespace change is good it should stay, though :) Probably as `(optional " ")` right after that closing parenthesis.